plugins/outputs/microsoft_fabric/event_house.go

```diff
@@ -0,0 +1,142 @@
+package microsoft_fabric
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-kusto-go/kusto/ingest"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/common/adx"
+	"github.com/influxdata/telegraf/plugins/serializers/json"
+)
+
+type eventhouse struct {
+	config *adx.Config
+	client *adx.Client
+
+	log        telegraf.Logger
+	serializer telegraf.Serializer
+}
+
+func (e *eventhouse) Init() error {
+	serializer := &json.Serializer{
+		TimestampUnits:  config.Duration(time.Nanosecond),
+		TimestampFormat: time.RFC3339Nano,
+	}
+	if err := serializer.Init(); err != nil {
+		return err
```

Comment:
Please use error prefixes!
```suggestion
		return fmt.Errorf("initializing JSON serializer failed: %w", err)
```